sematic/resolvers/local_resolver.py

```diff
@@ -222,6 +245,26 @@ def _cancel(data):
         self._create_resolution(self._root_future)
         self._update_resolution_status(ResolutionStatus.RUNNING)
 
+        self._cache_namespace = self._resolve_cache_namespace()
+
+    def _resolve_cache_namespace(self) -> Optional[str]:
```

Comment:
Can we maybe use a different word than "resolve" here to avoid confusion with the actual resolution? Maybe name it `_make_cache_namespace`?

Reply:
Done.